runtime/service.go

```diff
@@ -738,3 +754,30 @@ func packBundle(path string, options *ptypes.Any) (*ptypes.Any, error) {
 	}
 	return ptypes.MarshalAny(extraData)
 }
+
+// setupStubDrive will create a flat file with the drive ID as its contents.
+// The file is padded with null bytes to a minimum size of minFCDriveImageSize.
```

Comment:
It might be worth using a Pascal string (i.e., [length-prefixed string](https://en.wikipedia.org/wiki/String_(computer_science)#Length-prefixed)) or a [netstring](https://en.wikipedia.org/wiki/Netstring) to make the meaningful part of the file easy to reason about.